src/verification/deserialized_statement_verifier_v2.cpp

```diff
@@ -11,10 +11,26 @@ DeserializedStatementVerifierV2::DeserializedStatementVerifierV2(unique_ptr<SQLS
 unique_ptr<StatementVerifier> DeserializedStatementVerifierV2::Create(const SQLStatement &statement) {
 	auto &select_stmt = statement.Cast<SelectStatement>();
 
-	auto blob = BinarySerializer::Serialize(select_stmt);
+	auto blob = BinarySerializer::Serialize(select_stmt, true); // <-- Serialize default values
```

Comment:
```suggestion
	auto blob = BinarySerializer::Serialize(select_stmt, /*Serialize default values*/true);
```
I like this syntax to annotate parameters

Reply:
Alright, neither it is